account_invoice_supplier_number_info/README.rst

```diff
@@ -0,0 +1,39 @@
+ .. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+Invoice number:
+===============
+
+supplier_invoice_number field added in the tree and search view by
+inheritance
+
```

Comment:
You may want to add the 'try me on runbot' button https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst#usage